src/features/query-bar/components/filter/Condition.js

```diff
@@ -150,91 +156,122 @@ const Condition = ({
         const { valueKey, valueType } = condition;
 
         const isValueSet = valueKey !== null && valueKey !== '';
-
-        const message = valueType === 'date' ? messages.tooltipSelectDateError : messages.tooltipSelectValueError;
-
-        const error = areErrorsEnabled && !isValueSet ? formatMessage(message) : null;
+        const message = (
+            <FormattedMessage
+                {...(valueType === DATE ? messages.tooltipSelectDateError : messages.tooltipSelectValueError)}
+            />
+        );
+        const error = areErrorsEnabled && !isValueSet ? message : null;
 
         return error;
     };
 
     const renderDeleteButton = () => {
         return (
-            <div className="filter-item-delete-button">
+            <div className="condition-delete-button">
                 <button className="delete-button" onClick={onDeleteButtonClick} type="button">
                     <IconClose width={deleteButtonIconWidth} height={deleteButtonIconHeight} color="#999EA4" />
                 </button>
             </div>
         );
     };
 
-    const renderPrefixField = () => {
+    const renderConnectorField = () => {
+        let message = '';
```

Comment:
The name "message" confuses me. It's passed to "placeholder" but this is not the purpose of placeholders.